Resources/public/js/views/services/ez-universaldiscoveryviewservice.js

```diff
@@ -22,6 +22,53 @@ YUI.add('ez-universaldiscoveryviewservice', function (Y) {
      */
     Y.eZ.UniversalDiscoveryViewService = Y.Base.create('universalDiscoveryViewService', Y.eZ.ViewService, [Y.eZ.SideViewService], {
         /**
+         *
+         * @method _load
+         * @protected
+         * @param {Function} callback
+         */
+        _load: function (callback) {
+            var startingLocation = new Y.eZ.Location(),
+                path = [];
+
+            if (this.get('parameters').startingLocationId) {
+                startingLocation.set('id', this.get('parameters').startingLocationId);
+                this.get('app').set('loading', true);
+                this._loadStartingLocationPath(startingLocation, Y.bind(function() {
+                    startingLocation.get('path').unshift(this.get('virtualRootLocation'));
```

Comment:
you can not do that. `path` belongs to the Location and should not be modified from the outside. To avoid doing that, you should probably pass the virtual root Location to the UDW view in addition to the start location.